s3config.json

```diff
@@ -6,53 +6,451 @@
-    { "Condition": { "KeyPrefixEquals": "docs/" },                                                             "Redirect": { "HostName": "$REDIR_HOSTNAME", "ReplaceKeyPrefixWith": "", "HttpRedirectCode": "301" } },
-    { "Condition": { "KeyPrefixEquals": "pages/services/edge-lb/" },                                           "Redirect": { "HostName": "$REDIR_HOSTNAME", "ReplaceKeyPrefixWith": "services/edge-lb/", "HttpRedirectCode": "301" } },
-    { "Condition": { "KeyPrefixEquals": "pages/services/marathon-lb/" },                                       "Redirect": { "HostName": "$REDIR_HOSTNAME", "ReplaceKeyPrefixWith": "services/marathon-lb/services/edge-lb/", "HttpRedirectCode": "301" } },
-    { "Condition": { "KeyPrefixEquals": "services/kafka/configure/" },                                         "Redirect": { "HostName": "$REDIR_HOSTNAME", "ReplaceKeyPrefixWith": "services/kafka/kafka-auth/", "HttpRedirectCode": "301" } },
-    { "Condition": { "KeyPrefixEquals": "services/overview/" },                                                "Redirect": { "HostName": "$REDIR_HOSTNAME", "ReplaceKeyPrefixWith": "services/", "HttpRedirectCode": "301" } },
-    { "Condition": { "KeyPrefixEquals": "([0-9]+\\.[0-9]+/.*)$" },                                             "Redirect": { "HostName": "$REDIR_HOSTNAME", "ReplaceKeyPrefixWith": "mesosphere/dcos/", "HttpRedirectCode": "301" } },
-    { "Condition": { "KeyPrefixEquals": "services/" },                                                         "Redirect": { "HostName": "$REDIR_HOSTNAME", "ReplaceKeyPrefixWith": "mesosphere/dcos/services/", "HttpRedirectCode": "301" } },
-    { "Condition": { "KeyPrefixEquals": "version-policy/" },                                                   "Redirect": { "HostName": "$REDIR_HOSTNAME", "ReplaceKeyPrefixWith": "mesosphere/dcos/version-policy/", "HttpRedirectCode": "301" } },
-    { "Condition": { "KeyPrefixEquals": "support" },                                                           "Redirect": { "HostName": "support.d2iq.com", "Protocol": "https", "HttpRedirectCode": "301" } }
+    {
+      "Condition": { "KeyPrefixEquals": "mesosphere/dcos/latest/" },
+      "Redirect": {
+        "HostName": "$REDIR_HOSTNAME",
+        "ReplaceKeyPrefixWith": "mesosphere/dcos/2.2/",
+        "HttpRedirectCode": "307"
+      }
+    },
+    {
+      "Condition": { "KeyPrefixEquals": "dkp/konvoy/latest/" },
+      "Redirect": {
+        "HostName": "$REDIR_HOSTNAME",
+        "ReplaceKeyPrefixWith": "dkp/konvoy/2.2/",
+        "HttpRedirectCode": "307"
+      }
+    },
+    {
+      "Condition": { "KeyPrefixEquals": "dkp/dispatch/latest/" },
+      "Redirect": {
+        "HostName": "$REDIR_HOSTNAME",
+        "ReplaceKeyPrefixWith": "dkp/dispatch/1.4/",
+        "HttpRedirectCode": "307"
+      }
+    },
+    {
+      "Condition": {
+        "KeyPrefixEquals": "dkp/kommander/latest/operations/cloud-providers"
+      },
```

Comment:
Not critical, but the braces style for this condition is different than the previous ones and most of the remaining ones.

Reply:
It's totally valid!  I just thought you were going for 'consistent style'.   Like I said, not a big deal.